.mergify.yml

```diff
@@ -0,0 +1,39 @@
+---
+pull_request_rules:
+  - name: automatic merge
+    conditions:
+      - and: &base_checks
+        - base=master
+        - -label~=^acceptance-tests-needed|not-ready
+        # wait explicitly for one of the final checks to show up to be on the
+        # safe side even in case of checks not reporting back at all
+        - "check-success~=🐪 Perl"
+        # we don't need to spell out all tests but we can check a minimum
+        # amount
+        - "#check-success=>6"
```

Comment:
Can you make it, so that it's the entire workflow?. Checking, only a few doesn't sound too reassuring to me.

Reply:
Well, I don't need to include all as we also check that none failed. So we basically only need to cover any potential short "racy" time where some checks would have succeeded already but others did not yet even show up as pending. Likely this can not even happen as long as we only have github actions. This would e.g. only apply for something external like travisCI or circleCI which would report back with a pending status only after some seconds or even minutes